macros/edr/dbt_artifacts/upload_dbt_exposures.sql

```diff
@@ -58,7 +58,7 @@
         'package_name': node_dict.get('package_name'),
         'original_path': node_dict.get('original_file_path'),
         'path': node_dict.get('path'),
-        'generated_at': elementary.get_run_started_at().strftime('%Y-%m-%d %H:%M:%S')
+        'generated_at': modules.datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Maybe put this in a macro - 'current_timestamp_utc_as_string' (you can even get the format and set it to default '%Y-%m-%d %H:%M:%S'). The 'formatted_current_timestamp_column' could be current_timestamp_utc_as_timestamp.

Reply:
Moved the modules datetime to a macro.
Didn't rename the formatted_current_timestamp_column macro because this name describes it better.
This is a current timestamp column that is formatted. this is not a current timestamp as timestamp.